AttributeError: <config> object has no attribute 'pad_token_id' in transformers v5 convergence tests

Multiple convergence test failures because of uninitialized pad_token_id in multiple models in v5.

Example:
```
FAILED test/convergence/fp32/test_mini_models_with_logits.py::test_mini_model[mini_glm4v-32-0.0001-dtype15-1e-08-1e-05-0.005-1e-05-0.005-1e-05] - AttributeError: 'Glm4vTextConfig' object has no attribute 'pad_token_id'
FAILED test/convergence/fp32/test_mini_models_with_logits.py::test_mini_model[mini_exaone4-32-1e-05-dtype29-1e-08-1e-05-0.005-1e-05-0.005-1e-05] - AttributeError: 'Exaone4Config' object has no attribute 'pad_token_id'. Did you mean: 'bos_token_id
```

--- a/test/convergence/bf16/test_mini_models.py
+++ b/test/convergence/bf16/test_mini_models.py
@@ -857,6 +857,7 @@ if QWEN3_VL_AVAILABLE:
                 rms_norm_eps=1e-6,
                 use_cache=True,
                 vocab_size=32768,
+                pad_token_id=None,
                 rope_scaling=dict(
                     type="mrope",
                     mrope_section=[16, 24, 24],  # (temporal, height, width)
@@ -1164,6 +1165,7 @@ if GLM4V_AVAILABLE:
                     if not IS_TRANSFORMERS_V5_OR_LATER
                     else {}
                 ),
+                "pad_token_id": None,
             },
             vision_config={
                 "depth": 4,  # 32
@@ -1466,6 +1468,7 @@ if EXAONE4_AVAILABLE:
             use_cache=True,
             vocab_size=32000,
             attn_implementation="sdpa",
+            pad_token_id=None,
         ),
     )
 

--- a/test/convergence/bf16/test_mini_models_multimodal.py
+++ b/test/convergence/bf16/test_mini_models_multimodal.py
@@ -291,6 +291,7 @@ if LLAMA4_AVAILABLE:
                 vocab_size=32000,  # 128256,
             ),
             attn_implementation="sdpa",
+            pad_token_id=None,
         ),
     )
 
@@ -820,6 +821,7 @@ if QWEN3_VL_MOE_AVAILABLE:
                 num_experts_per_tok=2,
                 num_experts=4,
                 mlp_only_layers=[],
+                pad_token_id=None,
             ).to_dict(),
         ),
     )

--- a/test/convergence/bf16/test_mini_models_with_logits.py
+++ b/test/convergence/bf16/test_mini_models_with_logits.py
@@ -1109,6 +1109,7 @@ if GLM4V_AVAILABLE:
                     if not IS_TRANSFORMERS_V5_OR_LATER
                     else {}
                 ),
+                "pad_token_id": None,
             },
             vision_config={
                 "depth": 4,  # 32
@@ -1411,6 +1412,7 @@ if EXAONE4_AVAILABLE:
             use_cache=True,
             vocab_size=32000,
             attn_implementation="sdpa",
+            pad_token_id=None,
         ),
     )
 

--- a/test/convergence/fp32/test_mini_models.py
+++ b/test/convergence/fp32/test_mini_models.py
@@ -912,6 +912,7 @@ if QWEN3_VL_MOE_AVAILABLE:
                 num_experts=4,
                 tie_word_embeddings=False,
                 mlp_only_layers=[],
+                pad_token_id=None,
                 rope_scaling=dict(
                     type="mrope",
                     mrope_section=[16, 24, 24],  # (temporal, height, width)
@@ -1102,6 +1103,7 @@ if GLM4V_AVAILABLE:
                     if not IS_TRANSFORMERS_V5_OR_LATER
                     else {}
                 ),
+                "pad_token_id": None,
             },
             vision_config={
                 "depth": 4,  # 32
@@ -1454,6 +1456,7 @@ if EXAONE4_AVAILABLE:
             use_cache=True,
             vocab_size=32000,
             attn_implementation="sdpa",
+            pad_token_id=None,
         ),
     )
 

--- a/test/convergence/fp32/test_mini_models_with_logits.py
+++ b/test/convergence/fp32/test_mini_models_with_logits.py
@@ -1129,6 +1129,7 @@ if GLM4V_AVAILABLE:
                     if not IS_TRANSFORMERS_V5_OR_LATER
                     else {}
                 ),
+                "pad_token_id": None,
             },
             vision_config={
                 "depth": 4,  # 32
@@ -1426,6 +1427,7 @@ if EXAONE4_AVAILABLE:
             use_cache=True,
             vocab_size=32000,
             attn_implementation="sdpa",
+            pad_token_id=None,
         ),
     )
 

